mne/preprocessing/_lof.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+# Authors: Velu Prabhakar Kumaravel <vkumaravel@fbk.eu>
+# License: BSD-3-Clause
+
+
+import numpy as np
+from sklearn.neighbors import LocalOutlierFactor
```

Comment:
@vpKumaravel you need to import this inside the function. sklearn is only an dependency of mne.